directdemod/constants.py

```diff
@@ -43,6 +44,12 @@
 MAP_TEMPLATE = MODULE_PATH + "/misc/map.html"
 GLOBE_TEMPLATE = MODULE_PATH + "/misc/globe.html"
 
+## SSH settings
```

Comment:
What is this SSH for?

Reply:
These are special settings for ssh connection. The SSH connection is used to transmit data from the processing server to the webserver. There are two ways to upload images to the webserver:
1. Upload via a web site.
2. Upload via ssh.

Reply:
So basically there are two use cases:
1. Set of recordings stations transmit data to the processing server, that processes data and then sends it to the webserver via ssh.
2. Simply upload data via a web browser.